data/part-1/5-calculating.md

```diff
@@ -783,15 +783,15 @@ The last example produces an incorrectly rounded result, because the integer div
 
 <!-- Jos jakolausekkeen tulos asetetaan kokonaislukutyyppiseen muuttujaan, on tulos automaattisesti kokonaisluku. -->
 
-If the result of a division is assigned to an integer-type variable, the result is automatically an integer.
```

Comment:
statement is missing context information -- what kind of numbers are divided, double/float or integers.
Result CAN NOT be "automatically an integer" WITHOUT explicit conversion operator -> (int). Why?

§ Well, _the same issue_ we have here, _again_:<br />
![1-5e1 incompatible types (from double to int)!](https://user-images.githubusercontent.com/24639417/156840577-ec88cf8d-b1cd-427e-92b6-f35c0448d0ed.png)<br />
![1-5e2 Cast 3 0div2 to int!](https://user-images.githubusercontent.com/24639417/156840592-5902e9fd-ff7f-406f-bec6-57ed44812d32.png)<br />
![1-5e3 Uncompilable source code!](https://user-images.githubusercontent.com/24639417/156840597-15945a98-0a65-4c9e-80a9-e7df86c08243.png)

**_Is the result of division automatically an integer if it is assigned to an integer-type variable???_**
_YES or NO? The question is NOW!_ :grinning: 
<br />